lib/source-maps.js

```diff
@@ -0,0 +1,56 @@
+var convertSourceMap = require('convert-source-map')
+var libCoverage = require('istanbul-lib-coverage')
+var libSourceMaps = require('istanbul-lib-source-maps')
+var fs = require('fs')
+var path = require('path')
```

Comment:
You have `const`s elsewhere, but `var`s here. I'm guessing we're going to Node v4+ soon so I'd like to be consistent :)